contracts/Core/AssetManager.sol

```diff
@@ -82,37 +81,28 @@ contract AssetManager is AssetStorage, StateManager, AssetManagerParams, IAssetM
         uint32 epoch = _getEpoch(epochLength);
         if (assetStatus) {
             if (!collections[id].active) {
-                activeCollections.push(id);
-                collections[id].assetIndex = uint16(activeCollections.length);
+                // slither-disable-next-line incorrect-equality
```

Comment:
Are lines like L79 needed anymore, now we are not sharing same space for collection and jobs, so I think it can be removed or else it can be replaced by id <= numCollections

Reply:
I'll replace it with id <= numCollections